.vitepress/theme/style/index.scss

```diff
@@ -6,7 +6,7 @@ $iroha-brand: #e4232d;
   --code-font-family: 'JetBrains Mono', 'Fira Code', var(--font-family-mono);
   --code-font-size: 13px;
   --code-line-height: 1.2em;
-  --font-family-base: sans-serif;
+  --font-family-base: 'Source Serif 4';
```

Comment:
We probably want to use `Sora`, so that we remanin cohesive with the rest of Soramitsu's offerings.